cirq/contrib/json_test.py

```diff
@@ -12,9 +11,10 @@ def test_quantum_volume():
         heavy_set=[1, 2, 3],
         compiled_circuit=cirq.Circuit(cirq.H.on_each(qubits)),
         sampler_result=.1)
-    assert_json_roundtrip_works(qvr, resolvers=DEFAULT_CONTRIB_RESOLVERS)
+
+    assert_json_roundtrip_works(qvr)
```

Comment:
This test should still require you to explicitly say you're including unstable serialization data.

Reply:
The new model makes this unnecessary. Contrib is registered now as part of the `DEFAULT_RESOLVERS`. 

Also, as contrib is referenced from within `cirq/__init__.py`, it will not be able to rely on `DEFAULT_RESOLVERS` in a constant at package initialization to create `DEFAULT_CONTRIB_RESOLVERS`, because we can't guarantee that `DEFAULT_RESOLVERS` is "ready" by the time contrib is imported. 

We could unregister contrib potentially and rely on recreating a `contrib-resolvers` within each test but that doesn't seem to be a useful pattern for end users, as they would have to the same in their code.

Reply:
I'm very uncomfortable include contrib in the default resolvers. Contrib code is unstable, it shouldn't be included in the definition of a guaranteed-to-be-stable file format.

Reply:
OK, I'll think about this a bit more before merging.

Reply:
The root of the problem was that contrib is imported from cirq proper. After moving contrib to after the resolver cache registration, the tests pass now.